esp/src/eclwatch/PreflightDetailsWidget.js

```diff
@@ -47,95 +50,185 @@ define([
                 this.openButton = registry.byId(this.id + "Open");
                 this.openButton.set("disabled", true);
 
-                var retVal = new declare([ESPUtil.Grid(true, true)])({
+                var retVal = new declare([ESPUtil.Grid(true, false, {rowsPerPage:1000}, true)])({
                     store: this.store,
-                    columns: context.setColumns()
+                    columns: this.setColumns()
                 }, domID);
                 return retVal;
             },
 
+            calculateColumnWidth: function (grid) {
+                object.forIn(grid.columns, function (item, idx) {
```

Comment:
Is this the same a for (var key in grid) type of thing?  If so, lets just do it long hand?